apps/authentications/src/app/controllers/authentications.controller.v2.ts

```diff
@@ -48,4 +51,17 @@ export class AuthenticationControllerV2 {
       password
     );
   }
+
+  @ApiOkResponse({
+    type: CheckingResponse,
+  })
+  @Post('exists/castcle-id')
+  @HttpCode(200)
+  async checkCastcleIdExists(@Body() body: CheckIdExistDto) {
+    try {
+      return await this.authenticationService.checkUserExist(body.castcleId);
+    } catch (error) {
+      throw CastcleException.SOMETHING_WRONG;
+    }
```

Comment:
```suggestion
    return this.authenticationService.checkUserExist(body.castcleId);
```

มี exception filter อยู่แล้วนะ ไม่จำเป็นต้อง try catch ครอบส่วน controller ก็ได้ครับ

Reply:
+1